lottie-ios/Classes/Private/LOTAnimationView.m

```diff
@@ -190,6 +190,9 @@ - (void)_restoreState {
 - (void)_removeCurrentAnimationIfNecessary {
   _isAnimationPlaying = NO;
   [_compContainer removeAllAnimations];
+  if (_shouldRasterizeWhenIdle) {
+    _compContainer.shouldRasterize = YES;
```

Comment:
If Im not mistaken this will override the ShouldRasterizeWhenIdlebehavior whenever the animation stops. Shouldnt this be:
`_compContainer.shouldRasterize = _shouldRasterizeWhenIdle`

Reply:
@buba447  Not sure I fully understand what you mean by override the ShouldRasterizeWhenIdle behavior…

The idea was to rasterize the layer at the time when animation stops if and only if `shouldRasterizeWhenIdle == YES`. If `shouldRasterizeWhenIdle == NO` then `LOTAnimationView` will not try to mess with `_compContainer`'s `shouldRasterize` at all (and will potentially let the client control mess with it freely).

Reply:
That said, I guess it might indeed be more consistent and easy to follow if we just did `_compContainer.shouldRasterize = _shouldRasterizeWhenIdle` here and removed the `if (_shouldRasterizeWhenIdle) {` check in `playFromFrame:` and just set `shouldRasterize = NO` there unconditionally.

Reply:
Thats all i meant! Thanks!